core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -275,5 +280,16 @@ oppia.controller('CollectionPlayer', [
         $scope.generatePathParameters();
       }
     }, true);
+
+    $scope.scrollToLocation = function(id) {
+      $location.hash(id);
+      $anchorScroll();
+    };
+    // Touching anywhere outside the mobile preview should hide it.
```

Comment:
I think this confusion is because the "preview card" in the variable previewCardIsShown refers to the blank card and not the lesson card itself in the desktop version (ie. When previewCardIsShown is true, it means the blank card is shown and lesson card is hidden, that is why I've used `ng-if="!previewCardIsShown"` in the exploration-summary-tile tag) . Whereas, in the mobile view there is no preview card (blank card) and so it doesn't make sense in the context of the mobile view to have previewCardIsShown.

But now consider that the variable is renamed to previewCardIsHidden, only difference is that here the "preview card" refers to the active lesson card on both the desktop as well as the mobile version:
so when true, lesson card is hidden and when false, lesson card is shown in both the mobile and desktop versions.